### ISSUE TITLE
[Suggestion]: Buff Watcher for others

### ISSUE THREAD

**Issue by Sythalin** [Zeffuro/ZeffUI#68]
### What's your suggestion?

Ability to track other users' CDs.  Example:  When other casters' Swiftcast is on CD.

**Comment by Zeffuro** [comment]
This is in Party Cooldowns unless you mean something else?

**Comment by Sythalin** [comment]
> 

Nope, just a noob and just realized that was it.  Disregard, sorry!

**Comment by Sythalin** [comment]
Okay, relooked at it.  The icons are great, but looking for the ability to set up timer bars.  This is currently how I have it set with SPESPE:

https://i.imgur.com/yI5w6BZ.png

**Comment by Zeffuro** [comment]
Yeah currently not have it set up like that, would need a rewrite too prob
